src/scripts/towns/Town.ts

```diff
@@ -639,22 +683,40 @@ const HoennBerryMaster = new Shop([
 ]);
 
 //Hoenn NPCs
+
 const LittlerootAide = new NPC('Professor Birch\'s Aide', [
     'We have received word from Mr. Pokémon in Johto! He has made another breakthrough.',
     'You can now store an additional four eggs in the queue! His research has really gained speed.',
     'He wants you to know that he will have an additional eight slots ready by the time you reach Sinnoh.',
 ]);
 
-const SootopolisWallace = new NPC('Gym Leader Wallace', [
-    'The creators of the lands and ocean slumber within the Cave of Origin.',
-    'However, they will only awaken when in the presence of a truly great trainer.',
-    'You will have to overcome the Pokémon League before you have any chance to encounter them.',
+const OldaleTrackingScientist = new NPC('Tracking Scientist', [
+    'Tracking Scientist - Hey trainer, look at these footprints! Huh, I’ve never seen footprints like these before…They look like they came from two different Pokémon, and I saw two blurs, one red and one blue, quickly fly away just as I exited the Pokémon Center.',
```

Comment:
Doesn't need the name as already shows.
```suggestion
    'Hey trainer, look at these footprints! Huh, I’ve never seen footprints like these before…They look like they came from two different Pokémon, and I saw two blurs, one red and one blue, quickly fly away just as I exited the Pokémon Center.',
```